oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+Specification
+=============
+
+We believe that we can create a healthy and empowering dialogue around our architectural process and mitigate the above concerns by focusing on a re-alignment of ceremonies and clarifying the spirit and process around the creation and review of ADRs.
+
+We will adjust our current architectural process by:
+
+- Formalizing and clarifying an **Architectural Advisory Process** in which we conversationally provide input on and promptly review the status of ADRs, as well as review updates to our architectural principles and tech radar, with the goal of enabling more effective, informed decision making.
+- Improve the discoverability of stakeholders and experts via a **decision maker's checklist**.
+- Documenting as a *suggested best practice* a quarterly **technical strategic review process** where we can review our technical strategy and cross-functional requirements as high-level, guiding input to the Advisory Process, and to help teams and organizations decide where to prioritize addressing "important but not urgent" work such as architectural investments, new capabilities, addressing technical debt, and performing ongoing maintenance/upgrades.
+
+Architectural Advisory Process
+------------------------------
+
+We'll create an Architecture Advisory Process to add structure to today's informal review of ADRs.  The goal of this process is to review ADRs with relevant stakeholders and experts.  The process conversation needs to take place promptly so that decisions do not feel blocked on gathering advice.
+
+We intend to also use the Advisory Process over time to revisit and iterate on our architectural principles with the goal of ensuring they're specific, measurable, achievable, realistic, and testable. (a variation of SMART criteria)  Similarly, like ADRs and architectural principles, we can use the process to review and adjust Open edX Tech Radar blips.
+
+See `Appendix A: Advisory Process Principles <#Appendix A: Advisory Process Principles>`__  for more details on the spirit and functioning of the Advisory Process.
+
+Advisory Process Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The final artifact of the Advisory Process is an Architectural Decision Record document checked in to a repository relevant to the decision.  Prior to that, however, we suggest as a best practice using Github's Issues and Pull Request tooling to conduct the Advisory Process.  The advantage of doing so is that Issue/Pull Request comments remove the burden of recording who gave particular advice, and allow us to have threaded, focused conversations about various aspects of ADR.
```

Comment:
This part confuses me, but maybe it is because I am not familiar enough with the Github flow.  

Previous paragraph states "We'll create an Architecture Advisory Process to add structure to today's informal review of ADRs.  The goal of this process is to review ADRs with relevant stakeholders and experts."

However, this paragraph makes it seem like the ADR doesnt exist until the end of the review process, and there are only Issues and PRs before.  What exactly is the flow being proposed here...  Issue --> PR --> ADR?